pkg/client/unversioned/deployment.go

```
@@ -36,6 +36,7 @@ type DeploymentInterface interface {
 	Update(*extensions.Deployment) (*extensions.Deployment, error)
 	UpdateStatus(*extensions.Deployment) (*extensions.Deployment, error)
 	Watch(opts api.ListOptions) (watch.Interface, error)
+	Rollback(*extensions.DeploymentRollback) (*extensions.DeploymentRollback, error)
```

Fixed (it should just return the error since it's POST).